.ado/windows-vs-pr.yml

```diff
@@ -227,7 +227,7 @@ jobs:
       - template: templates/prepare-env.yml
         parameters:
           applyRnPatches: $(applyRnPatches)
```

Comment:
@jonthysell I removed all of these recently, but it looks like your recent change incorporated them before I removed them. Would you be able to remove all of the instances of applyRnPatches?

Reply:
yes, I clean that up in https://github.com/microsoft/react-native-windows/pull/4285

Reply:
Thanks! https://github.com/microsoft/react-native-windows/commit/0489bcdae52eb2223983813923929f046521f797#diff-73b804cbbf6b019df7af5e2760333758 is the PR removing them for reference